---
layout: paper
title: "Protein tolerance to random circular permutation correlates with thermostability and local energetics of residue-residue contacts"
authors: Atkinson JT, Jones AM, Nanda V, and Silberg JJ
year: 2019
ref: Atkinson et al. 2019. Protein Eng. Des. Sel.
journal: "Protein Engineering, Design, & Selection"
issue: "32(11): 489-501 <br> <b><i> Editor's Choice</i></b>"
doi: 10.1093/protein/gzaa012
image: /images/papers/2019_PEDScover.jpeg
pdf: /pdfs/papers/2020_Atkinson JT_Protein tolerance to random circular permutation correlates with thermostability and local energetics of residue-residue contacts.pdf
---

## Abstract

Adenylate kinase (AK) orthologs with a range of thermostabilities were subjected to random circular permutation, and deep mutational scanning was used to evaluate where new protein termini were nondisruptive to activity. The fraction of circularly permuted variants that retained function in each library correlated with AK thermostability. In addition, analysis of the positional tolerance to new termini, which increase local conformational flexibility, showed that bonds were either functionally sensitive to cleavage across all homologs, differentially sensitive, or uniformly tolerant. The mobile AMP-binding domain, which displays the highest calculated contact energies, presented the greatest tolerance to new termini across all AKs. In contrast, retention of function in the lid and core domains was more dependent upon AK melting temperature. These results show that family permutation profiling identifies primary structure that has been selected by evolution for dynamics that are critical to activity within an enzyme family. These findings also illustrate how deep mutational scanning can be applied to protein homologs in parallel to differentiate how topology, stability, and local energetics govern mutational tolerance.